package/koreader/package

```diff
@@ -5,10 +5,10 @@
 pkgnames=(koreader)
 pkgdesc="Ebook reader supporting PDF, DjVu, EPUB, FB2 and many more formats"
 url=https://github.com/koreader/koreader
-pkgver=2021.03-1
-timestamp=2021-03-28T11:28:48Z
+pkgver=2021.04-1
+timestamp=2021-04-22T06:29:20Z
 section="readers"
-maintainer="raisjn <of.raisjn@gmail.com>"
+maintainer="Eeems <eeems@eeems.email>"
```

Comment:
i've thought about it a bit and i think i'll stay maintainer. the maintainer's job is to accept changes / review PRs which i think i've been doing. if you become maintainer and then put up a PRs, i think the process will be slower than if i am maintainer